modin/experimental/core/execution/native/implementations/hdk_on_native/calcite_builder.py

```diff
@@ -419,7 +449,6 @@ class InputContext:
             "min": "MIN",
             "size": "COUNT",
             "count": "COUNT",
-            "median": "APPROX_QUANTILE",
```

Comment:
Why?

Reply:
Now quantile is used for median - https://github.com/modin-project/modin/pull/6528/files#diff-d1a0db76b871ca475d604ff109cc3ebf13d7ead8ce07cca7caa54ace5f736cc0R706